Update netatmo.markdown (#20598)
home-assistant/home-assistant.github.io: source/_integrations/netatmo.markdown

  type: string
{% endconfiguration %}

> **After the client_id and client_secret is added to your `configuration.yaml` you must restart Home Assistant en then enable Netatmo through the integrations page.**

Reviewer: ```suggestion
**After the `client_id` and `client_secret` are added to your `configuration.yaml` you must restart Home Assistant and then enable Netatmo through the integrations page.**
```
